theano/compile/profilemode.py

```diff
@@ -693,6 +693,22 @@ def exp_float32_op(op):
                           " generator supported on the GPU.")
                 break
 
+        # tip 6
+        import theano.sandbox.cuda as cuda
+        from theano.tensor.nnet import LogSoftmax
+        import theano.tensor.signal.pool as pool
+
+        for a, t in iteritems(apply_time):
+            node = a[1]
+            if (isinstance(node.op, pool.Pool)):
+                if not cuda.dnn.dnn_available():
```

Comment:
Also do the check for the new back-end
import theano.gpuarray.dnn
if not not cuda.dnn.dnn_available() or not theano.gpuarray.dnn.dnn_present()

idem bellow.